### PR TITLE
Merging DM concepts etc. from WebProtégé

### PR DESCRIPTION
**Pull request by matuskalas**
TODO:
- [x] Definition of cybersecurity
- [x] ~Check indentation everywhere where suspicious~ (done in #884, merge after this)
- [x] Add back US label 'Data digitalization', as a skos:editorialNote for now (needs a fix in rOBOt, or moving the single-label check to Caséologue)

### CODE REVIEW

File: EDAM_dev.owl [hunk 1]
```diff
@@ -52325,6 +52325,56 @@
...
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://edamontology.org/has_topic"/>
+                <owl:someValuesFrom rdf:resource="http://edamontology.org/topic_4011"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <created_in>1.26</created_in>
+        <oboInOwl:created_by>bianchini</oboInOwl:created_by>
+        <oboInOwl:creation_date>2023-03-02T12:02:42.14696Z</oboInOwl:creation_date>
+        <oboInOwl:hasDefinition></oboInOwl:hasDefinition>
```
Comment (matuskalas) [inline EDAM_dev.owl:52371]:
```suggestion
        <oboInOwl:hasDefinition>.</oboInOwl:hasDefinition>
```

**Review by matuskalas — COMMENTED** [review]
I think this is great, we just need to define Data digitalisation and ideally add some Wikimedia links

**Comment by bianchini88** [comment]
> we just need to define Data digitalisation and ideally add some Wikimedia links

Done, hopefully.

**Comment by bianchini88** [comment]
@matuskalas debugged, hopefully.

**Comment by matuskalas** [comment]
❤ Million thanks for fixing these @bianchini88 !
And million thanks Caséologue folks for having this awesome CI ! ❤ @LucieLamothe @maanst @hmenager @albangaignard

**Comment by bianchini88** [comment]
@matuskalas I re-triggered the failing jobs (both here and on https://github.com/edamontology/edamontology/pull/884) and they worked this time.

File: EDAM_dev.owl [hunk 2]
```diff
@@ -56796,19 +56849,27 @@
...
         <oboInOwl:hasDbXref>VT 1.3.1 Data management</oboInOwl:hasDbXref>
-        <oboInOwl:hasDefinition>Data management comprises the practices and principles of taking care of data, other than analysing them. This includes for example taking care of the associated metadata, formatting, storage, archiving, or access.</oboInOwl:hasDefinition>
+        <oboInOwl:hasDefinition>Data management comprises the practices and principles of taking care of data, other than analysing them. This includes for example taking care of the associated metadata, data formats, storage, preservation/archiving, access, and legal aspects related to using and sharing data.</oboInOwl:hasDefinition>
+        <oboInOwl:hasNarrowSynonym>Metadata management</oboInOwl:hasNarrowSynonym>
+        <oboInOwl:hasNarrowSynonym>Research data management (RDM)</oboInOwl:hasNarrowSynonym>
         <oboInOwl:inSubset rdf:resource="http://edamontology.org/bio"/>
         <oboInOwl:inSubset rdf:resource="http://edamontology.org/events"/>
         <oboInOwl:inSubset rdf:resource="http://edamontology.org/topics"/>
+        <oboInOwl:savedBy>bianchini</oboInOwl:savedBy>
```
Comment (matuskalas) [inline EDAM_dev.owl:56861]:
```suggestion
        <oboInOwl:savedBy>bianchini</oboInOwl:savedBy>
        <oboInOwl:savedBy>erin.calhoun</oboInOwl:savedBy>
```

**Review by matuskalas — COMMENTED** [review]
All awesome, let's do the final review & polishing the definitions

File: EDAM_dev.owl [hunk 3]
```diff
@@ -52325,6 +52326,58 @@
...
+                <owl:someValuesFrom rdf:resource="http://edamontology.org/topic_3071"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <created_in>1.26</created_in>
+        <oboInOwl:created_by>bianchini</oboInOwl:created_by>
+        <oboInOwl:creation_date>2023-03-02</oboInOwl:creation_date>
+        <oboInOwl:hasDefinition>Describe the operations that you will be performing to acquire, manage, document, store, share, and preserve your data. This includes budgeting for these operation when required.</oboInOwl:hasDefinition>
+        <oboInOwl:inSubset rdf:resource="http://edamontology.org/bio"/>
+        <oboInOwl:inSubset rdf:resource="http://edamontology.org/operations"/>
+        <rdfs:label>Data management planning</rdfs:label>
```
Comment (matuskalas) [inline EDAM_dev.owl:52354]:
```suggestion
        <oboInOwl:savedBy>bianchini</oboInOwl:savedBy>
        <oboInOwl:savedBy>Matúš Kalaš</oboInOwl:savedBy>
        <rdfs:comment>The purpose of data management planning is ... avoiding extra costs.</rdfs:comment>
        <rdfs:label>Data management planning</rdfs:label>
```

File: EDAM_dev.owl [hunk 4]
```diff
@@ -52325,6 +52326,58 @@
...
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://edamontology.org/has_topic"/>
+                <owl:someValuesFrom rdf:resource="http://edamontology.org/topic_3071"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <created_in>1.26</created_in>
+        <oboInOwl:created_by>bianchini</oboInOwl:created_by>
+        <oboInOwl:creation_date>2023-03-02</oboInOwl:creation_date>
+        <oboInOwl:hasDefinition>Describe the operations that you will be performing to acquire, manage, document, store, share, and preserve your data. This includes budgeting for these operation when required.</oboInOwl:hasDefinition>
```
Comment (matuskalas) [inline EDAM_dev.owl:52351]:
```suggestion
        <oboInOwl:hasDefinition>Data management planning is the process of creating and updating a data management plan for a project or an organisation.</oboInOwl:hasDefinition>
```

File: EDAM_dev.owl [hunk 5]
```diff
@@ -52325,6 +52326,62 @@
...
+        <oboInOwl:created_by>bianchini</oboInOwl:created_by>
+        <oboInOwl:creation_date>2023-03-02T12:02:42.14696Z</oboInOwl:creation_date>
+        <oboInOwl:hasDefinition>Digitisation is the process of converting data into a machine-readable format.</oboInOwl:hasDefinition>
+        <oboInOwl:hasNarrowSynonym>Metadata recording</oboInOwl:hasNarrowSynonym>
+        <oboInOwl:inSubset rdf:resource="http://edamontology.org/bio"/>
+        <oboInOwl:inSubset rdf:resource="http://edamontology.org/operations"/>
+	<skos:exactMatch rdf:resource="https://www.wikidata.org/wiki/Q843958"/>
+	<rdfs:seeAlso rdf:resource="http://en.wikipedia.org/wiki/Digitization"/>
+        <rdfs:label>Data digitalisation</rdfs:label>
+        <skos:editorialNote xml:lang="en-us">Data digitalization</skos:editorialNote>
```
Comment (matuskalas) [inline EDAM_dev.owl:52383]:
```suggestion
        <skos:editorialNote xml:lang="en-us">Data digitization</skos:editorialNote>
```

File: EDAM_dev.owl [hunk 6]
```diff
@@ -52325,6 +52326,62 @@
...
+        <created_in>1.26</created_in>
+        <oboInOwl:created_by>bianchini</oboInOwl:created_by>
+        <oboInOwl:creation_date>2023-03-02T12:02:42.14696Z</oboInOwl:creation_date>
+        <oboInOwl:hasDefinition>Digitisation is the process of converting data into a machine-readable format.</oboInOwl:hasDefinition>
+        <oboInOwl:hasNarrowSynonym>Metadata recording</oboInOwl:hasNarrowSynonym>
+        <oboInOwl:inSubset rdf:resource="http://edamontology.org/bio"/>
+        <oboInOwl:inSubset rdf:resource="http://edamontology.org/operations"/>
+	<skos:exactMatch rdf:resource="https://www.wikidata.org/wiki/Q843958"/>
+	<rdfs:seeAlso rdf:resource="http://en.wikipedia.org/wiki/Digitization"/>
+        <rdfs:label>Data digitalisation</rdfs:label>
```
Comment (matuskalas) [inline EDAM_dev.owl:52382]:
```suggestion
        <rdfs:label>Data digitisation</rdfs:label>
```

File: EDAM_dev.owl [hunk 7]
```diff
@@ -52325,6 +52326,62 @@
...
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://edamontology.org/has_topic"/>
+                <owl:someValuesFrom rdf:resource="http://edamontology.org/topic_4011"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <created_in>1.26</created_in>
+        <oboInOwl:created_by>bianchini</oboInOwl:created_by>
+        <oboInOwl:creation_date>2023-03-02T12:02:42.14696Z</oboInOwl:creation_date>
+        <oboInOwl:hasDefinition>Digitisation is the process of converting data into a machine-readable format.</oboInOwl:hasDefinition>
+        <oboInOwl:hasNarrowSynonym>Metadata recording</oboInOwl:hasNarrowSynonym>
```
Comment (matuskalas) [inline EDAM_dev.owl:52377]:
```suggestion
        <oboInOwl:hasNarrowSynonym>Digital reformatting</oboInOwl:hasNarrowSynonym>
```

File: EDAM_dev.owl [hunk 8]
```diff
@@ -52325,6 +52326,62 @@
...
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://edamontology.org/has_topic"/>
+                <owl:someValuesFrom rdf:resource="http://edamontology.org/topic_4011"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <created_in>1.26</created_in>
+        <oboInOwl:created_by>bianchini</oboInOwl:created_by>
+        <oboInOwl:creation_date>2023-03-02T12:02:42.14696Z</oboInOwl:creation_date>
+        <oboInOwl:hasDefinition>Digitisation is the process of converting data into a machine-readable format.</oboInOwl:hasDefinition>
```
Comment (matuskalas) [inline EDAM_dev.owl:52376]:
```suggestion
        <oboInOwl:hasDefinition>Data digitisation is the process of converting analog data, text, or information into a digital format, appropriate for computational analysis and preservation.</oboInOwl:hasDefinition>
        <oboInOwl:hasExactSynonym>Data digitalisation</oboInOwl:hasExactSynonym>
        <oboInOwl:hasExactSynonym xml:lang="en-us">Data digitalization</oboInOwl:hasExactSynonym>
```

**Pull request merged**